packages/ipfs/src/core/components/pin/pin-manager.js

```diff
@@ -1,32 +1,30 @@
 /* eslint max-nested-callbacks: ["error", 8] */
 'use strict'
 
-const { DAGNode, DAGLink } = require('ipld-dag-pb')
 const CID = require('cids')
-const { default: Queue } = require('p-queue')
-const { Key } = require('interface-datastore')
 const errCode = require('err-code')
-const multicodec = require('multicodec')
 const dagCborLinks = require('dag-cbor-links')
 const debug = require('debug')
-const { Buffer } = require('buffer')
-const { cidToString } = require('../../../utils/cid')
-
-const createPinSet = require('./pin-set')
-
-const { Errors } = require('interface-datastore')
-const ERR_NOT_FOUND = Errors.notFoundError().code
+// const parallelBatch = require('it-parallel-batch')
+const first = require('it-first')
+const all = require('it-all')
+const cbor = require('cbor')
+const multibase = require('multibase')
 
 // arbitrary limit to the number of concurrent dag operations
-const WALK_DAG_CONCURRENCY_LIMIT = 300
-const IS_PINNED_WITH_TYPE_CONCURRENCY_LIMIT = 300
-const PIN_DS_KEY = new Key('/local/pins')
+// const WALK_DAG_CONCURRENCY_LIMIT = 300
+// const IS_PINNED_WITH_TYPE_CONCURRENCY_LIMIT = 300
+// const PIN_DS_KEY = new Key('/local/pins')
 
 function invalidPinTypeErr (type) {
   const errMsg = `Invalid type '${type}', must be one of {direct, indirect, recursive, all}`
   return errCode(new Error(errMsg), 'ERR_INVALID_PIN_TYPE')
 }
 
+function toKey (cid) {
+  return '/' + multibase.encode('base32', cid.multihash).toString().substring(1)
```

Comment:
Why strip out the multibase encoding? Seems like it could come in handy when working with CID tooling. Is it not worth the extra stored character?

Reply:
No particular reason, it was just what was being done elsewhere with datastore keys.

Reply:
No strong feelings here, but for what it's worth I'm planning on doing more logging of datastore keys with multibase in go-ipfs since it'll make it easier to compare/track CIDs, peerIDs, etc. as they move through the system.